sqlboiler/graph/templates/singleton/helper.tpl

```diff
@@ -56,12 +56,12 @@ func QueryModPagination(sinceID *graphql.ID, pageNum, pageSize *int32) []qm.Quer
 // QueryModSearch returns a list of search QueryMod based on the struct values
 func QueryModSearch(input interface{}) []qm.QueryMod {
   mods := []qm.QueryMod{}
-  if reflect.ValueOf(input).IsNil() {
+  if input == nil {
     return mods
   }
   // Get reflect value
   v := reflect.ValueOf(input).Elem()
-  if !v.IsValid() || v.IsNil() {
```

Comment:
this was added in https://github.com/choonkeat/hellocrud/commit/e66eac9dd0b159bc53e21e1d86ef341450006069#diff-a3e23cbdf329a381fbbbb9fce2aaa1f6

> if it's trying to evaluate a non-nilable data type.

then, what happens when it is trying to evaluate nillable data type?

Reply:
Was thinking `input/v` should be a `struct`? If it's nilable, it'll be caught by the `if input == nil`

Reply:
Actually, the problem I encountered is due to the `v.IsNil()` since `v` is no longer a `ptr` due to `v := reflect.ValueOf(input).Elem()`. This causes it to panic on `v.IsNil()` call. In this PR, I removed this check since it should be handled by the `== nil` prior to this line.

Reply:
👌 added tests! please 👀